File: practice_algos/javascript/algo22.js
// 22. Add Seven - Build a function that accepts an array. Return a new array
// with all the values of the original, but add 7 to each. Do not alter the
// original array.  Example, addSeven([1,2,3]) should return [8,9,10] in a new array

function addSeven(arr) {
    var tempArr = []
    for (var i = 0; i < arr.length; i++) {
        tempArr[i] = arr[i] + 7
    }
    return tempArr
}

console.log(addSeven([1,2,3]))

File: practice_algos/python/algo22.py
# 22. Add Seven - Build a function that accepts an array. Return a new array
# with all the values of the original, but add 7 to each. Do not alter the
# original array.  Example, addSeven([1,2,3]) should return [8,9,10] in a
# new array

def add_seven(arr):
    temp_arr = []
    for i in range(len(arr)):
        temp_arr.append(arr[i] + 7)
    return temp_arr

print(add_seven([1,2,3]))